spec.md

```diff
@@ -705,14 +704,17 @@ The response format is as follows:
 
 ### Listing Repositories
 
-Images are stored in collections, known as a _repository_, which is keyed by a `name`, as seen throughout the API specification.
-A registry instance MAY contain several repositories.
-The list of available repositories is made available through the _catalog_.
+Repository listing is reserved for a future version of the distribution spec.
+
+The `_catalog` api is reserved for historical usage. Registries MAY implement _catalog, but are NOT required.
```

Comment:
Can we link to Docker v2 registry protocol spec in reference to this.
Because as of right now there is absolutely no context about what `_catalog` _was_

Reply:
We discussed on the call this week. Rather than have a pointer for someone to implement an old, inconsistently implemented spec, we agreed to focus on defining a new api. The intent here isn't to say here's a deprecated api. Rather, we're stating `_catalog` isn't part of the spec, however we do reserve it so a future version of the spec wouldn't attempt to use it. Thus, preserving any previous implementations.

Reply:
Thanks Jimmy,
Are you ok if we don’t reference it? You’ve got this mares as requesting the change. 
I’m sure there are many places to get the info. Just keeping the spec as clean as possible.

Reply:
I'm fine with not including it because it keeps the spec more self contained and will only become more irrelevant over time.